odl/solvers/functional/default_functionals.py

```diff
@@ -464,3 +464,67 @@ def __init__(self, space):
             Domain of the functional.
         """
         super().__init__(space=space, constant=0)
+
+
+class IndicatorBox(Functional):
+
+    """Indicator on some box shaped domain."""
```

Comment:
Should be clear in what sense this is a box (i.e. in the VALUES) of the incoming function.